blockchain/processing.py

```diff
@@ -368,7 +368,7 @@ def check_tx_sig_existence(self, transaction):
         valid = True
         if not transaction.tx_signature:
             valid = False
-        elif not transaction.tx_signature.digest:
```

Comment:
Make sure in future to reset branch to master before starting another ticket/branch...
:)